test_kata.py

```diff
@@ -9,5 +9,11 @@ def test_empty_string(self):
     def test_two_nums(self):
         self.assertEqual(Add('2,4'), 6)
 
+    def test_4_nums(self):
```

Comment:
You've switched from using `two` (as in, spell the number out with letters) to numeric characters `4` and `10` here, which I would normally assume is a sign that you've context switched / taken a break and come back 😄 

That's super subtle and extremely minor, but as the old adage goes [naming is one of the two hard problems in computer science](https://martinfowler.com/bliki/TwoHardThings.html).